Makefile

```diff
@@ -1,12 +1,12 @@
 #/***************************************************************************
-# SPAQLunicorn
+# SpaLOD
 #
-# This plugin adds a GeoJSON layer from a Wikidata SPARQL query.
+# Fork of SPARQLunicorn
 #							 -------------------
-#		begin				: 2019-10-28
+#		begin				: 2022-02-02
 #		git sha				: $Format:%H$
-#		copyright			: (C) 2019 by SPARQL Unicorn
-#		email				: rse@fthiery.de
+#		copyright			: (C) 2022 by HAFSAOUI A, PONCIANO C , PONCIANO JJ
+#		email				: antoinehafsaoui@gmail.com
```

Comment:
You cannot void someone else's copyright and take it over simply by forking a repo and making what appear (after partial review) to be relatively minor changes and additions. Most of this remain © SPARQL Unicorn with email <rse@fthiery.de>.